services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -9,6 +9,7 @@
 import uuid as uuidlib
 from datetime import datetime
 from enum import Enum
```

Comment:
MINOR: run ``make install-dev autoformat`` in ``web/server`` from time to time so that imports get sorted

Reply:
👍 I really wanted to know how to do this!